### DISCUSSION TITLE
Apply matching quotes and back ticks?

### DISCUSSION THREAD

**Discussion by marlonjames71** [coteditor/CotEditor#1734] [General]
Is there a setting that makes it possible to apply matching quotes, single quotes, angle brackets, and back ticks? I see CotEditor has matching square brackets, curly braces, and parens to selected text, but it doesn't work with the things I just mentioned previously (quotes, etc.).

Thanks.

**Comment by 1024jp** [comment]
Could you elaborate a bit more on what you mean by "match"?

**Reply by marlonjames71** [reply]
For example, if I select some text and type ", then the selected text would have opening and closing quotes applied to it. As this should work with other matching delimiters as well:

https://github.com/user-attachments/assets/19a01c9b-f947-4311-a111-683a89c16b02
